modules/create_timepoint/jsx/createTimepointIndex.js

```diff
@@ -0,0 +1,358 @@
+    if (!this.state.isLoaded) {
+      return <Loader/>;
+    }
+
+    /**
+     * Populate the elements of errors to display.
+     * @param {array} values (errors) data from server.
+     * @return {array} individual errors.
+     */
+    function renderErrors(values) {
+      let errors = [];
+      Object.keys(values).forEach(function(key) {
+        errors.push(
+          <div className='col-xs-12 col-sm-12 col-md-12'>
+            <label className='error form-group'>
+              {values[key]}
+            </label>
+          </div>
+        );
+      });
+      return errors;
+    }
+
+    // Include form errors.
+    const errors = this.state.errors
+      ? renderErrors(this.state.errors)
+      : null;
+    // Include subproject select.
+    const subproject = this.state.form.display.subproject ? (
+      <SelectElement
+        id={'subproject'}
+        name={'subproject'}
+        label={'Subproject'}
+        value={this.state.form.value.subproject}
+        options={this.state.form.options.subproject}
+        onUserInput={this.setForm}
+        emptyOption={true}
+        disabled={false}
+        required={this.state.form.options.required.subproject}
+      />
+    ) : null;
+    // Include psc select.
+    const psc = this.state.form.display.psc ? (
+      <SelectElement
+        id={'psc'}
+        name={'psc'}
+        label={'Site'}
+        value={this.state.form.value.psc}
+        options={this.state.form.options.psc}
+        onUserInput={this.setForm}
+        emptyOption={true}
+        disabled={false}
+        required={this.state.form.options.required.psc}
+      />
+    ) : null;
+    // Include visit select.
+    const visit = this.state.form.display.visit ? (
+      <SelectElement
+        id={'visit'}
+        name={'visit'}
+        label={'Visit label'}
+        value={this.state.form.value.visit}
+        options={this.state.form.options.visit}
+        onUserInput={this.setForm}
+        emptyOption={true}
+        disabled={false}
+        required={this.state.form.options.required.visit}
+      />
+    ) : null;
+    // Include languages select.
+    const languages = this.state.form.display.languages ? (
+      <SelectElement
+        id={'languageID'}
+        name={'languages'}
+        label={'Language'}
+        value={this.state.form.value.languages}
+        options={this.state.form.options.languages}
+        onUserInput={this.setForm}
+        emptyOption={true}
+        disabled={false}
+        required={this.state.form.options.required.languages}
+      />
+    ) : null;
+
+    return (
+      <Panel title='Create Time Point'>
+        {errors}
+        <FormElement
+          name={'timepointInfo'}
+          class={'form-group col-sm-12'}
+          onSubmit={this.handleSubmit}
+        >
+          <StaticElement
+            label={'DCCID'}
+            text={this.state.data.dccid}
+          />
+          {subproject}
+          {psc}
+          {visit}
```

Comment:
on master (screenshot below), the create timepoint form has a Project field as well that doesn't appear here. not sure if this is intentional but the PR description made no mention of removing it

<img width="541" alt="Screen Shot 2020-07-07 at 6 31 46 PM" src="https://user-images.githubusercontent.com/17415878/86851663-58b82a00-c081-11ea-8130-be6733e46dc4.png">

Reply:
thanks @zaliqarosli, I'm not sure what happened to it. I'll fix.